android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java

```diff
@@ -759,4 +769,18 @@ private static boolean shouldSkip() {
     }
     return false;
   }
+
+  class SafeMethodChannelInterceptor implements ClientInterceptor {
+    @Override
+    public <ReqT, RespT> ClientCall<ReqT, RespT> interceptCall(MethodDescriptor<ReqT, RespT> method,
+        CallOptions callOptions, Channel next) {
+      return new CheckedForwardingClientCall<ReqT, RespT>(next.newCall(
+          method.toBuilder().setSafe(true).build(), callOptions)) {
+        @Override
+        public void checkedStart(final Listener<RespT> responseListener, Metadata headers) {
```

Comment:
optional: I would remove final.  It doesn't add any value and it clutters the code.